### ISSUE TITLE
I need help with 'Pull Request'

### ISSUE THREAD

**Issue by XanoXeno727** [githubschool/open-enrollment-classes-introduction-to-github#17009]
My pull requests failed test and I am unable to confirm with merge button like the video says..?
What should I do to solve this problem?

**Comment by XanoXeno727** [comment]
Yeah I have same problem with same failed results. I thought maybe the servers on github might be overloaded or something, but that would not make sense either because nobody would be able to access github site either lol. 
Followed the instructional video exactly how it was done so I'm not sure what corrections I need to make??

**Comment by jomustech** [comment]
Tried again today, same thing, I asked for help from a gitubteacher, mentioned you have the same problem, hopefully it will be solved. 😄

**Comment by XanoXeno727** [comment]
Yeah I'm also looking into other online resources to solve it. This might only be related to a very small issue solving it should not be difficult.

**Comment by jomustech** [comment]
So true, my mistake was a misspelling. 
Thanks to the teacher who helped :)

**Comment by jomustech** [comment]
That was by accident, ok.

**Comment by Edditoria** [comment]
Hello @XanoXeno727 . Have read your commit d7d6e29 .

First, please read the point (2) in the tutorial page [Work on GitHub](https://services.github.com/on-demand/intro-to-github/work-on-github)

```
---
githubHandle: YOUR-USERNAME
latitude: YOUR_LATITUDE
longitude: YOUR_LONGITUDE
---
```

Then, please compare to your own code. You will find that you are missing one line of code.

This is a good chance to understand how important to read documentations very carefully. BTW, you are close!! Go! 🏎 ~~

**Comment by XanoXeno727** [comment]
Hello @Edditoria. 
Which line of code am I missing exactly? Is it the last 3 dashes of code? I compared my own code with the [Work on GitHub](https://services.github.com/on-demand/intro-to-github/work-on-github) code and the only code I seem to be missing is the last 3 dashes.

As the video tutorial suggested, I assigned the pull request to myself along with the coordinates via NASA website and there are 2 check errors (from Travis CI build) and 2 successful checks (from me) following the Merge button on the original Pull Request I made. 
It does not allow me to access the Merge button to complete the pull request I sent. 

I also noticed there is an option to submit the [Merge] from the [Pull Request] on the command line.
This would be a more direct way to solve the errors I received previously, right? 
Or do I still need one more line of code before I can Merge it?

**Comment by Edditoria** [comment]
Yes. It is missing the last 3 dasher `---`. I assume you are going to complete it in **existing** pull request #17008 . So, in real situation, you need to:

1. Go to the page of your pull request. ([link](https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/17008))
2. Click the **latest** commit link. (named: "Add XanoXeno727's pin to the map") ([link](https://github.com/githubschool/open-enrollment-classes-introduction-to-github/pull/17008/commits/d7d6e299508fa3119c8f7b93d44ba1ff51756ae4))
3. Click the edit button (at right hand side) ([link](https://github.com/githubschool/open-enrollment-classes-introduction-to-github/edit/XanoXeno727-northcarolina/_pins/XanoXeno727.json?pr=/githubschool/open-enrollment-classes-introduction-to-github/pull/17008))
3. Commit it.

Then, your pull request #17008 will list one more commit. The CI will check it again.

Remember, you need to make sure that you are editing the file in the branch you want, and in the PR you want. The above processes can make sure it.

Good luck. Happy Coding!!

**Comment by MatheusRV** [comment]
Heyy you must follow tutorial on https://services.github.com/on-demand/intro-to-github/
